observer/observer.go

```diff
@@ -0,0 +1,73 @@
+package observer
+
+import (
+	blog "github.com/letsencrypt/boulder/log"
+	// _ are probes imported to trigger init func
+	_ "github.com/letsencrypt/boulder/observer/probes/dns"
+	_ "github.com/letsencrypt/boulder/observer/probes/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	statTotalMonitors = prometheus.NewCounterVec(
```

Comment:
The style we use elsewhere (and which was mandatory via Google Style) is that all counter metrics be named "fooCount".